OpenChain Telco SBOM Specification.md

```diff
@@ -169,4 +169,18 @@ Some open source software licenses enable any recipient to redistribute the soft
 ## 4. Conformant notice
 To indicate that the software has a conformant SBOM available, you may use the following statement: “This software is supplied with a SBOM conformant to the OpenChain Telco SBOM specification v1.0, available at https://github.com/OpenChain-Project/Telco-WG/blob/main/OpenChain%20Telco%20SBOM%20Specification.md”
 
-You may at your choosing use the following statement in your Telco SBOM “This SBOM conforms to the OpenChain Telco SBOM specification v1.0, available at https://github.com/OpenChain-Project/Telco-WG/blob/main/OpenChain%20Telco%20SBOM%20Specification.md, it is provided to the recipient free of charge, and the recipient is free to redistribute this SBOM to any third party that they distribute the corresponding software to, provided that they have all the necessary right to distribute the software to such third party”
+You may at your choosing use the following statement in you Telco Speficatation conformant SBOM “This SBOM conforms to the OpenChain Telco SBOM specification v1.0 [Insert homepage], it is provided to the recipient free of charge, and the recipient is free to redistribute this SBOM to any 3rd that they distribute the corresponding software to, provided that they have all the necessary right to distribute the software to such 3rd party”
```

Comment:
Quite a few typos in this line. I suggest:

You may at your choosing use the following statement in your Telco Specification-conformant SBOM: “This SBOM conforms to the OpenChain Telco SBOM specification v1.0 [Insert homepage], it is provided to the recipient free of charge, and the recipient is free to redistribute this SBOM to any third party to whom they distribute the corresponding software, provided that they have all the necessary rights to distribute the software to such third party.”

What's the reason for dropping the link to the homepage _and replacing it with a placeholder_?